tumblr_client.gemspec

```diff
@@ -4,14 +4,14 @@ require File.join(File.dirname(__FILE__), 'lib/tumblr/version')
 Gem::Specification.new do |gem|
   gem.add_dependency 'faraday', '~> 0.9.0'
```

Comment:
Can the dependency on `faraday` be updated from `~> 0.9.0` to `~> 0.9` while at it, in line with other dependencies?  There's already faraday 0.10.0, which should be upper compatible with 0.9.x.